modules/ROOT/pages/blogs/entry/announce-apache-netbeans-23-released.adoc

```diff
@@ -0,0 +1,55 @@
+// 
+//     Licensed to the Apache Software Foundation (ASF) under one
+//     or more contributor license agreements.  See the NOTICE file
+//     distributed with this work for additional information
+//     regarding copyright ownership.  The ASF licenses this file
+//     to you under the Apache License, Version 2.0 (the
+//     "License"); you may not use this file except in compliance
+//     with the License.  You may obtain a copy of the License at
+// 
+//       http://www.apache.org/licenses/LICENSE-2.0
+// 
+//     Unless required by applicable law or agreed to in writing,
+//     software distributed under the License is distributed on an
+//     "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//     KIND, either express or implied.  See the License for the
+//     specific language governing permissions and limitations
+//     under the License.
+//
+
+= [ANNOUNCE] Apache NetBeans 23 Released
+:author: Geertjan Wielenga
+:page-revdate: 2024-09-19
+:page-layout: blogentry
+:page-tags: blogentry
+:jbake-status: published
+:keywords: Apache NetBeans 23 release
+:description: Apache NetBeans 23 release
+:toc: left
+:toc-title:
+:page-syntax: true
+
+
+The Apache NetBeans team is pleased to announce that Apache NetBeans 23 was released on September 19, 2024.
+
+What's in the Apache NetBeans 23 release:
+
+https://github.com/apache/netbeans/releases/tag/23
+
+With thanks to 22 contributors, including 3 who have contributed for the first time. Thank you!
```

Comment:
its 24 contributors according to the release notes page

Reply:
done